report/src/problem_analysis/hci/designing_products.tex

```diff
@@ -0,0 +1,16 @@
+\section{Designing a product} % generel hci
+When establishing a design for a product, one often begins with exploring initial thoughts and ideas for a design, only to then present the design to the user, make adjustments, and repeat the process. 
+This can be described in a process called the design funnel.\cite{BUXTON2007135_skething}
+
+Buxton\cite{BUXTON2007135_skething,BUXTON2007139_prototyping} describes how to move from an exploratory analysis of different designs through sketching to interactive, product refinement using prototypes. 
+Sketching is the act of ideation and capturing key concepts of the design in a cheap and easy-to-make artifact. 
+It is important to notice that the artifact itself is but a byproduct of the process, while the goal of the activity lies within the process itself.\cite{BUXTON2007135_skething}
+The true goal is to understand user needs and develop a design concept that both the designer and user find agreeable in terms of requirements and user experience. 
+When sketching, one must put aside assumptions and preconceptions about the product one is designing.
+This helps the designer discover new ideas for the product due to not being constrained by an existing understanding of requirements, user needs, and expert knowledge. %kilde? 
```

Comment:
Either you cite or you don't comment that you need a cite. No todos!

```suggestion
This helps the designer discover new ideas for the product due to not being constrained by an existing understanding of requirements, user needs, and expert knowledge.
```

Reply:
```suggestion
This helps discovering new ideas for the product from not being constrained by an existing understanding of requirements, user needs, and expert knowledge.
```